src/js/angular/clustermanagement/templates/modal/remove-nodes-dialog.html

```diff
@@ -0,0 +1,44 @@
+<!-- Start remove nodes from cluster modal -->
+
+<div class="modal-header">
+    <button type="button" class="close" ng-click="cancel()"></button>
+    <h3 class="modal-title">{{'cluster_management.remove_nodes_dialog.title' | translate}}</h3>
+</div>
+<form ng-submit="ok()" name="form">
+    <div class="modal-body">
+        <h5>{{'cluster_management.remove_nodes_dialog.content' | translate}}</h5>
+        <div class="row">
+            <div class="col-xs-12 mb-1 mt-1">
+                <h5>{{'cluster_management.cluster_page.cluster_nodes_list' | translate}}</h5>
+                <ul ng-if="clusterNodes.length" class="list-group nodes-list">
+                    <li ng-repeat="node in clusterNodes" ng-class="{'list-group-item-danger': node.shouldRemove}"
+                        class="list-group-item list-group-item-action d-flex justify-content-between align-items-center node">
+                        <div class="location-item">
+                            <div class="logo-image"></div>
+                            {{node.endpoint}}
+                        </div>
+                        <button type="button" ng-click="toggleNode($index, node)" class="btn btn-link toggle-node-btn">
+                            <span ng-class="node.shouldRemove ? 'icon-check' : 'icon-minus'"></span>
+                        </button>
+                    </li>
+                </ul>
+                <div class="row">
+                    <div class="col-xs-12">
+                        <div ng-if="clusterNodes.length - nodesToRemoveCount < 2" class="not-enough-nodes-error alert alert-danger mt-1">
+                            {{'cluster_management.cluster_page.errors.at_least_two_nodes' | translate}}
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" ng-click="cancel()">{{'common.cancel.btn' | translate}}</button>
+        <button id="wb-remove-nodes-from-cluster-submit" class="btn btn-primary"
+                ng-disabled="!nodesToRemoveCount || (clusterNodes.length - nodesToRemoveCount < 2)"
```

Comment:
Can you expose a variable in the controller for this `clusterNodes.length - nodesToRemoveCount < 2` and bind to it, then reuse it few lines above.